src/libraries/System.Private.CoreLib/src/System/Decimal.DecCalc.cs

```diff
@@ -196,7 +196,7 @@ private static void UInt64x64To128(ulong a, ulong b, ref DecCalc result)
                     high++;
 
                 if (high > uint.MaxValue)
-                    Number.ThrowOverflowException(TypeCode.Decimal);
+                    Number.ThrowOverflowException(SR.Overflow_Decimal);
```

Comment:
Should at least use `Number.ThrowOverflowException<decimal>();` here if this is going to be changed.

Reply:
This wasn't resolved, should I open a separate PR for this?